crates/config/src/etherscan.rs

```diff
@@ -173,6 +174,9 @@ pub struct EtherscanConfig {
     /// Etherscan API URL
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub url: Option<String>,
+    /// Etherscan API Version. Defaults to v2
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub api_version: Option<EtherscanApiVersion>,
```

Comment:
we could probably remove the Option here here if we make EtherscanApiVersion::V2 the default with the explorer pr?

Reply:
there's one scenario I want to test, we provide support for custom verifiers compatible with Etherscan APIs which probably won't update to v2 style at the same time (or never?) So will make sure it doesn't brake

Reply:
I think the option is important in the case that we actually want to say it’s up to the backend to choose the version. 

Good idea for a new test case.

Reply:
makes sense